README.md

```diff
@@ -1,24 +1,30 @@
 # Testable Combine Publishers
 
-An easy, declarative way to unit test Combine Publishers in Swift
+An easy, declarative way to unit test Combine Publishers in Swift. Available for both XCTest and Swift Testing.
 
-![Example Combine Unit Test](example.png)
+**XCTest**:
+![Example Combine Unit Test - XCTest](example_xctest.png)
+
+**Swift Testing**:
+![Example Combine Unit Test - Swift Testing](example_swift_testing.png)
```

Comment:
Formatting nit: The text and image show up inline with each other unless there's an extra newline.

```suggestion
**Swift Testing**:

![Example Combine Unit Test - Swift Testing](example_swift_testing.png)
```